applications/ShapeOptimizationApplication/custom_python/shape_optimization_python_application.cpp

```diff
@@ -49,9 +49,9 @@ PYBIND11_MODULE(KratosShapeOptimizationApplication, m)
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, NORMALIZED_SURFACE_NORMAL);
 
     // Optimization variables
-    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DF1DX);
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DFDX);
```

Comment:
I changed it back to stay compatible. I originally just changed the name because of the former implementation concept. As another concept was realized, this change is not needed anymore.